File: problem02/Euler2.java
public class Euler2 {

	public Euler2() {
	}

	public static void main(String[] args) {
		int sum = 0;
		int f = 0;
		int n = 0;

		while(f < 4000000) {
			n++;
			f = fib(n);
			if(f%2 == 0) {
				sum = sum + f;
			}
		}
		System.out.println(sum);
	}

	public static int fib(int n) {

		if(n <= 2) {
			return 1;
		} else {
			return fib(n-1) + fib(n-2);
		}
	}
}

File: problem02/Euler2.py
def iEven(num):
    return num%2==0

def fib(num):
    f = [0,1]
    for i in range(2,num+1):
        f.append(f[i-1] + f[i-2])
    return f[num]

sum = 0
f = 0
n = 0

while f < 4000000:
    n += 1
    f = fib(n)
    if f%2==0:
        sum = sum + f

print sum
